Author: Jamie Strandboge <jamie@canonical.com>
Description: allow owner 'rw' access to /{,var/}run/user/guest-*/dconf/user.
 Also allow owner write of /{,var/}run/user/guest-*/keyring-*/
Bug-Ubuntu: https://launchpad.net/bugs/1059510

Index: lightdm-1.3.3/data/guest-session.apparmor
===================================================================
--- lightdm-1.3.3.orig/data/guest-session.apparmor	2012-10-01 10:15:36.000000000 -0500
+++ lightdm-1.3.3/data/guest-session.apparmor	2012-10-01 10:24:12.000000000 -0500
@@ -58,6 +58,11 @@
   # necessary for writing to sockets, etc.
   /{,var/}run/** rmkix,
   /{,var/}run/shm/** wl,
+  # libpam-xdg-support
+  owner /{,var/}run/user/guest-*/dconf/ rw,
+  owner /{,var/}run/user/guest-*/dconf/user rw,
+  owner /{,var/}run/user/guest-*/keyring-*/ rw,
+  owner /{,var/}run/user/guest-*/keyring-*/{control,gpg,pkcs11,ssh} rw,
 
   capability ipc_lock,
 
